_sections/events/watm-festival/2020-06-12-2130-dance-like-nobodys-watching.md

```diff
@@ -0,0 +1,16 @@
+---
+type: watm-event
+title: dance-like-nobodys-watching
+name: Dance Like Nobody's Watching
+facilitator: Cameron Payne
+day: 1
+date: '2020-06-12'
+start-time: '21:30'
+end-time: '23:30'
+link: TBA
+calendar-link:
+---
+
+Tap into your "inner child". Remember when you were a child, dancing and bouncing around on your parent's bed? This dance party will be broadcast from my bedroom and my bed. I will take musical requests via chat. Let’s dance like nobody's watching.
+
+Cameron Fairfax Payne age: 50, Profession: tie-dye artist for 35 years, Recovery time; 1 year, diagnosed with ADHD at 5 years old. prescribed stimulant therapy until age 20. Was in active addiction from about 13 years old. Seek Healing and the network of facilitators has helped me with direction and support in my healing path.
```

Comment:
SeekHealing one word, right?

Reply:
Eagle Eyes over here! I am but a humble copy/paster, tending to my reams of text.